script/clean

```diff
@@ -39,12 +40,32 @@ var run = function() {
 
   if (Array.isArray(next)) {
     var pathToRemove = path.resolve.apply(path.resolve, next);
-    if (fs.existsSync(pathToRemove))
-      next = removeCommand + pathToRemove;
-    else
+    if (fs.existsSync(pathToRemove)) {
+      if (isWindows) {
+        removeFolderRecursive(pathToRemove);
+      } else {
+        next = removeCommand + pathToRemove;
+        cp.safeExec(next, run);
+      }
+    }
+    else {
       return run();
+    }
   }
-
-  cp.safeExec(next, run);
+  else
+    cp.safeExec(next, run);
 };
 run();
+
+// Windows has a 260-char path limit for rmdir etc. Just recursively delete in Node.
```

Comment:
The big reason to use rimraf is that it uses `graceful-fs` behind the scenes, so it automatically retries transient ACCESS_DENIEDs that Windows AV vendors love to throw